contracts/inputs/InputBox.sol

```diff
@@ -1,7 +1,7 @@
 // (c) Cartesi and individual authors (see AUTHORS)
 // SPDX-License-Identifier: Apache-2.0 (see LICENSE)
 
-pragma solidity ^0.8.13;
+pragma solidity ^0.8.18;
```

Comment:
The `block.prevrandao` global variable was introduced in [0.8.18](https://github.com/ethereum/solidity/releases/tag/v0.8.18).